src/routes/Login/Form/Form.index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { string } from "prop-types";
+import styles from "./Form.module.css";
+
+import EmailSvg from "../../../../public/assets/Message.svg";
+import PasswordSvg from "../../../../public/assets/Lock.svg";
+
+export default function Form({ children, type }) {
+  const { labelSvg, formContainer, label } = styles;
+  const icon = {
+    email: <EmailSvg className={labelSvg} />,
+    password: <PasswordSvg className={labelSvg} />,
+  };
+
+  return (
+    <div>
```

Comment:
Sorry forgot to add. This is also useless div, because we can use label instead of that div. Main rule of react is that it require only 1 parent element, and in this case it can be label